x/account/keeper/msg_server_create_coordinator_test.go

```diff
@@ -0,0 +1,79 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/x/account/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestMsgCreateCoordinator(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  types.MsgCreateCoordinator
+		want uint64
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: types.MsgCreateCoordinator{
+				Address: "invalid_address",
+				Description: &types.CoordinatorDescription{
+					Identity: "identity_invalid_address",
+					Website:  "website_invalid_address",
+					Details:  "details_invalid_address",
+				},
+			},
+			err: status.Error(codes.InvalidArgument,
+				"invalid coordinator address (invalid_address): decoding bech32 failed: invalid index of 1"),
+		}, {
+			name: "valid coordinator 1",
+			msg: types.MsgCreateCoordinator{
```

Comment:
We can create a new file `sample/profile` to return samples of structure from the `profile` module (where here you would provide the address as a parameter and it would randomize the description) so it simplifies tests writing

Reply:
https://github.com/tendermint/spn/blob/feat/init-vested-account/testutil/sample/launch.go

Reply:
I agree, but for this tests, we can create an import cycle using the sample and returning a `MsgCreateCoordinator`

Reply:
This is not a blocker for this PR but we should try to use `package keeper_test` as much as possible for this problem

https://github.com/tendermint/spn/issues/191